airflow-core/src/airflow/api_fastapi/execution_api/routes/__init__.py

```diff
@@ -42,6 +44,7 @@
 authenticated_router.include_router(asset_events.router, prefix="/asset-events", tags=["Asset Events"])
 authenticated_router.include_router(connections.router, prefix="/connections", tags=["Connections"])
 authenticated_router.include_router(dag_runs.router, prefix="/dag-runs", tags=["Dag Runs"])
+#authenticated_router.include_router(dags.router, prefix="/dags", tags=["Dags"])
```

Comment:
Should this be removed or uncommented?

Reply:
I plan to uncomment it, and remove the change at line 38. Just that I'm not yet sure on the token approach while calling the API.